From e210ddbf7adb23243907745fc432fa7c62d95c27 Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Thu, 22 Jan 2015 18:02:50 +0800
Subject: [PATCH] board: remove Config.in.boardname file

We can put the board name configuration in Config.in, no need to
create seperate file Config.in.boardname as no option will select
in it.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
---
 board/Config.in                            | 36 +++++++++++++-----------------
 board/at91sam9260ek/Config.in.boardname    |  2 --
 board/at91sam9261ek/Config.in.boardname    |  2 --
 board/at91sam9263ek/Config.in.boardname    |  2 --
 board/at91sam9g10ek/Config.in.boardname    |  2 --
 board/at91sam9g20ek/Config.in.boardname    |  2 --
 board/at91sam9m10g45ek/Config.in.boardname |  2 --
 board/at91sam9n12ek/Config.in.boardname    |  2 --
 board/at91sam9rlek/Config.in.boardname     |  2 --
 board/at91sam9x5ek/Config.in.boardname     |  2 --
 board/at91sam9xeek/Config.in.boardname     |  2 --
 board/sama5d3_xplained/Config.in.boardname |  2 --
 board/sama5d3x_cmp/Config.in.boardname     |  2 --
 board/sama5d3xek/Config.in.boardname       |  2 --
 board/sama5d4_xplained/Config.in.boardname |  2 --
 board/sama5d4ek/Config.in.boardname        |  2 --
 16 files changed, 16 insertions(+), 50 deletions(-)
 delete mode 100644 board/at91sam9260ek/Config.in.boardname
 delete mode 100644 board/at91sam9261ek/Config.in.boardname
 delete mode 100644 board/at91sam9263ek/Config.in.boardname
 delete mode 100644 board/at91sam9g10ek/Config.in.boardname
 delete mode 100644 board/at91sam9g20ek/Config.in.boardname
 delete mode 100644 board/at91sam9m10g45ek/Config.in.boardname
 delete mode 100644 board/at91sam9n12ek/Config.in.boardname
 delete mode 100644 board/at91sam9rlek/Config.in.boardname
 delete mode 100644 board/at91sam9x5ek/Config.in.boardname
 delete mode 100644 board/at91sam9xeek/Config.in.boardname
 delete mode 100644 board/sama5d3_xplained/Config.in.boardname
 delete mode 100644 board/sama5d3x_cmp/Config.in.boardname
 delete mode 100644 board/sama5d3xek/Config.in.boardname
 delete mode 100644 board/sama5d4_xplained/Config.in.boardname
 delete mode 100644 board/sama5d4ek/Config.in.boardname

diff --git a/board/Config.in b/board/Config.in
index bc9fd13..760f87b 100644
--- a/board/Config.in
+++ b/board/Config.in
@@ -27,26 +27,22 @@ source "board/sama5d4_xplained/Config.in.board"
 endchoice
 
 config CONFIG_BOARDNAME
-	string "Board Name"
-	help
-	    Name of the board, A Board Support package
-	    (BSP) must be available.
-
-source "board/at91sam9260ek/Config.in.boardname"
-source "board/at91sam9261ek/Config.in.boardname"
-source "board/at91sam9263ek/Config.in.boardname"
-source "board/at91sam9rlek/Config.in.boardname"
-source "board/at91sam9xeek/Config.in.boardname"
-source "board/at91sam9g10ek/Config.in.boardname"
-source "board/at91sam9g20ek/Config.in.boardname"
-source "board/at91sam9m10g45ek/Config.in.boardname"
-source "board/at91sam9n12ek/Config.in.boardname"
-source "board/at91sam9x5ek/Config.in.boardname"
-source "board/sama5d3xek/Config.in.boardname"
-source "board/sama5d3_xplained/Config.in.boardname"
-source "board/sama5d3x_cmp/Config.in.boardname"
-source "board/sama5d4ek/Config.in.boardname"
-source "board/sama5d4_xplained/Config.in.boardname"
+	string
+	default "at91sam9rlek"	if CONFIG_AT91SAM9RLEK
+	default "at91sam9260ek"	if CONFIG_AT91SAM9260EK
+	default "at91sam9261ek"	if CONFIG_AT91SAM9261EK
+	default "at91sam9263ek"	if CONFIG_AT91SAM9263EK
+	default "at91sam9xeek"	if CONFIG_AT91SAM9XEEK
+	default "at91sam9g10ek"	if CONFIG_AT91SAM9G10EK
+	default "at91sam9g20ek"	if CONFIG_AT91SAM9G20EK
+	default "at91sam9m10g45ek" if CONFIG_AT91SAM9M10G45EK
+	default "at91sam9n12ek"	if CONFIG_AT91SAM9N12EK
+	default "at91sam9x5ek"	if CONFIG_AT91SAM9X5EK
+	default "sama5d3_xplained" if CONFIG_SAMA5D3_XPLAINED
+	default "sama5d3xek" if CONFIG_SAMA5D3XEK
+	default "sama5d3x_cmp" if CONFIG_SAMA5D3X_CMP
+	default "sama5d4_xplained" if CONFIG_SAMA5D4_XPLAINED
+	default "sama5d4ek" if CONFIG_SAMA5D4EK
 
 config AT91SAM9260
 	select CPU_HAS_TWI0
diff --git a/board/at91sam9260ek/Config.in.boardname b/board/at91sam9260ek/Config.in.boardname
deleted file mode 100644
index 24cd0db..0000000
--- a/board/at91sam9260ek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "at91sam9260ek"	if CONFIG_AT91SAM9260EK
diff --git a/board/at91sam9261ek/Config.in.boardname b/board/at91sam9261ek/Config.in.boardname
deleted file mode 100644
index a81ae89..0000000
--- a/board/at91sam9261ek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "at91sam9261ek"	if CONFIG_AT91SAM9261EK
diff --git a/board/at91sam9263ek/Config.in.boardname b/board/at91sam9263ek/Config.in.boardname
deleted file mode 100644
index 93bb02d..0000000
--- a/board/at91sam9263ek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "at91sam9263ek"	if CONFIG_AT91SAM9263EK
diff --git a/board/at91sam9g10ek/Config.in.boardname b/board/at91sam9g10ek/Config.in.boardname
deleted file mode 100644
index 94d6b10..0000000
--- a/board/at91sam9g10ek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "at91sam9g10ek"	if CONFIG_AT91SAM9G10EK
diff --git a/board/at91sam9g20ek/Config.in.boardname b/board/at91sam9g20ek/Config.in.boardname
deleted file mode 100644
index 7cf61f2..0000000
--- a/board/at91sam9g20ek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "at91sam9g20ek"	if CONFIG_AT91SAM9G20EK
diff --git a/board/at91sam9m10g45ek/Config.in.boardname b/board/at91sam9m10g45ek/Config.in.boardname
deleted file mode 100644
index 4f11d41..0000000
--- a/board/at91sam9m10g45ek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "at91sam9m10g45ek" if CONFIG_AT91SAM9M10G45EK
diff --git a/board/at91sam9n12ek/Config.in.boardname b/board/at91sam9n12ek/Config.in.boardname
deleted file mode 100644
index 0b4699a..0000000
--- a/board/at91sam9n12ek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "at91sam9n12ek"	if CONFIG_AT91SAM9N12EK
diff --git a/board/at91sam9rlek/Config.in.boardname b/board/at91sam9rlek/Config.in.boardname
deleted file mode 100644
index e79b48d..0000000
--- a/board/at91sam9rlek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "at91sam9rlek"	if CONFIG_AT91SAM9RLEK
diff --git a/board/at91sam9x5ek/Config.in.boardname b/board/at91sam9x5ek/Config.in.boardname
deleted file mode 100644
index fcaa282..0000000
--- a/board/at91sam9x5ek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "at91sam9x5ek"	if CONFIG_AT91SAM9X5EK
diff --git a/board/at91sam9xeek/Config.in.boardname b/board/at91sam9xeek/Config.in.boardname
deleted file mode 100644
index 1e696f8..0000000
--- a/board/at91sam9xeek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "at91sam9xeek"	if CONFIG_AT91SAM9XEEK
diff --git a/board/sama5d3_xplained/Config.in.boardname b/board/sama5d3_xplained/Config.in.boardname
deleted file mode 100644
index d5046a4..0000000
--- a/board/sama5d3_xplained/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "sama5d3_xplained" if CONFIG_SAMA5D3_XPLAINED
diff --git a/board/sama5d3x_cmp/Config.in.boardname b/board/sama5d3x_cmp/Config.in.boardname
deleted file mode 100644
index ee42850..0000000
--- a/board/sama5d3x_cmp/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "sama5d3x_cmp" if CONFIG_SAMA5D3X_CMP
diff --git a/board/sama5d3xek/Config.in.boardname b/board/sama5d3xek/Config.in.boardname
deleted file mode 100644
index 7771eaa..0000000
--- a/board/sama5d3xek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "sama5d3xek" if CONFIG_SAMA5D3XEK
diff --git a/board/sama5d4_xplained/Config.in.boardname b/board/sama5d4_xplained/Config.in.boardname
deleted file mode 100644
index d1c485a..0000000
--- a/board/sama5d4_xplained/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "sama5d4_xplained" if CONFIG_SAMA5D4_XPLAINED
diff --git a/board/sama5d4ek/Config.in.boardname b/board/sama5d4ek/Config.in.boardname
deleted file mode 100644
index 36f3057..0000000
--- a/board/sama5d4ek/Config.in.boardname
+++ /dev/null
@@ -1,2 +0,0 @@
-config CONFIG_BOARDNAME
-	default "sama5d4ek" if CONFIG_SAMA5D4EK
-- 
1.9.1

